main.cpp

```diff
@@ -467,17 +466,8 @@ int main(int argc, char* argv[]) {
 
     // Test text
     glColor3f(0.1f, 0.2f, 0.5f);
-    
-    
-    
-    std::string debuggingText =
-        std::string("boats: ") +
-            std::to_string(boats.size()) +
-        std::string("\nsparks: ") +
-            std::to_string(sparks.size()) +
-        std::string("\n\xB5/F: ") + //'\xB5' being the micro symbol
-            std::to_string(uspf.count());
-    text->draw(debuggingText);
+    //'\xB5' = µ = micro sign (in case your editor's font can't handle it)
```

Comment:
nit: Ideally this should have been in a variable: char symbolMicro = '\xB5'; or something.